aws/table_aws_accessanalyzer_finding.go

```diff
@@ -0,0 +1,281 @@
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Condition"),
+			},
+			{
+				Name:        "created_at",
+				Description: "The time at which the finding was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Finding.CreatedAt"),
+			},
+			{
+				Name:        "error",
+				Description: "The error that resulted in an Error finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Error"),
+			},
+			{
+				Name:        "is_public",
+				Description: "Indicates whether the finding reports a resource that has a policy that allows public access.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("Finding.IsPublic"),
+			},
+			{
+				Name:        "principal",
+				Description: "The external principal that has access to a resource within the zone of trust.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Principal"),
+			},
+			{
+				Name:        "resource",
+				Description: "The resource that the external principal has access to.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Resource"),
+			},
+			{
+				Name:        "resource_owner_account",
+				Description: "The Amazon Web Services account ID that owns the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.ResourceOwnerAccount"),
+			},
+			{
+				Name:        "resource_type",
+				Description: "The type of the resource that the external principal has access to.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.ResourceType"),
+			},
+			{
+				Name:        "sources",
+				Description: "The sources of the finding, indicating how the access that generated the finding is granted. It is populated for Amazon S3 bucket findings.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Sources"),
+			},
+			{
+				Name:        "status",
+				Description: "The status of the finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Status"),
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time at which the finding was most recently updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Finding.UpdatedAt"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Id"),
+			},
+			{
+				Name:        "tags",
+				Description: "A placeholder for tags, as findings typically do not have tags but are included for consistency.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromConstant(map[string]*string{}),
+			},
+			{
+				Name:        "akas",
```

Comment:
Remove the column `aka`, because we don't have any ARN for the Finding, it is the ARN of it's parent.